website/docs/modules/sources.html.markdown

```diff
@@ -360,6 +360,8 @@ module "consul" {
 }
 ```
 
+<em>Note:</em> If your bucket lives in us-east-1, omit the region in the path URL (```s3::https://s3.amazonaws.com/...```)
```

Comment:
This code span should use single backticks instead of triple backticks; the note could also benefit from our callout box convention for notes. How about: 

```suggestion
-> **Note:** Buckets in AWS's us-east-1 region must use the hostname `s3.amazonaws.com` (instead of `s3-us-east-1.amazonaws.com`).
```

(The arrow is a non-standard Markdown extension used in our docs. It creates a blue callout box for notes that ought to stand out a bit.)